department-of-veterans-affairs/va.gov-cms
# VACMS-2043: Disable deprecated fields for Connect with us

## Description

Relates to #7168 .

## Testing done
Edited a CLP and saw the old field for connect with us was removed. Viewed the section taxonomy and saw the old connect with us fields were removed.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/144931034-563238ce-72c9-4736-b4b9-a00c560c52a0.png)
![image](https://user-images.githubusercontent.com/8375335/144931179-fac9c8b9-093f-43cb-a34c-805a19528004.png)


## QA steps

As a cms editor I can
1. edit a campaign landing page (node/15051/edit) and see the old fields removed
1. view the section taxonomy manage fields (/admin/structure/taxonomy/manage/administration/overview/fields) and see there are only two fields (description and product)
1. Then validate Acceptance Criteria from issue
- [x]  Delete social media link field, connect with us link field and email updates link text and url from section taxonomy.
- [x] Delete the 'add a connect with us panel' from CLP

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
